src/__internal__/input/input-presentation.style.ts

```diff
@@ -69,19 +69,23 @@ const InputPresentationStyle = styled.div<
       | "warning"
       | "info"
       | "hasIcon"
+      | "borderRadius"
     > &
     Pick<CarbonProviderProps, "validationRedesignOptIn">
 >`
   align-items: stretch;
   background: var(--colorsUtilityYang100);
   border: 1px solid var(--colorsUtilityMajor300);
-  border-radius: var(--borderRadius050);
   box-sizing: border-box;
   cursor: text;
   display: flex;
   flex-wrap: wrap;
   width: 100%;
   margin: 0;
+
+  ${({ borderRadius }) =>
+    `border-radius: ${borderRadius || "var(--borderRadius050)"};`}
```

Comment:
**suggestion** (non-blocking): this default `--borderRadius050` could be set as a default prop to `InputPresentation`, which would remove the need to have it here with the `||`. I think it would also make it more obvious to readers of the code what the default is (it should also make the default show up in the props table on Storybook, although I just realised that this is an internal component so that's not relevant here!)

Reply:
Good suggestion. That didn't occur to me to do it that way. I'm OK with making that change as it's good to make it obvious.